File: IT/Stringhe/Esercizi/Java/Esercizio4.java
/* file Esercizio4.java */

public class Esercizio4 {
	public static void main(String [] args) {
		String s = "Ciao, mondo!";

		for (int i = s.length() - 1; i >= 0; i--) {
			System.out.print(s.charAt(i));
		}

		System.out.print(System.lineSeparator());
	}
}

File: IT/Stringhe/Esercizi/C++/esercizio4.cpp
/* file esercizio4.cpp */

#include <iostream>

using std::string;
using std::cout;
using std::endl;

int main()
{
	string s = "Ciao, mondo!";

	for (int i = s.size() - 1; i >= 0; i--) {
		cout << s[i];
	}

	cout << endl;
}

